### PR TITLE
google-sheets: add basic chart test

### PR DESCRIPTION
**Pull request by mmelko**
`@add-chart-basic-chart-to-spreadsheet`

### CODE REVIEW

**Comment by syndesisqe** [comment]
Build finished. Tests PASSED

File: ui-tests/src/test/resources/features/integrations/google-sheets.feature [hunk 1]
```diff
@@ -374,9 +374,44 @@ Feature: Google Sheets Connector
...
+      | spreadsheetid | 1_OLTcj_y8NwST9KHhg8etB10xr6t3TrzaFXwW2dhpXw |
+    Then click on the "Done" button
+
+    When select the "google-sheets" connection
+    And select "Add charts" integration action
+    Then click on the "Done" button
+
+    When add integration step on position "0"
+    And select the "Data Mapper" connection
+    And check visibility of data mapper ui
```
Comment (avano) [inline ui-tests/src/test/resources/features/integrations/google-sheets.feature]: Then

File: ui-tests/src/test/resources/features/integrations/google-sheets.feature [hunk 2]
```diff
@@ -374,9 +374,44 @@ Feature: Google Sheets Connector
...
+    And define property "sheetId" with value "789378340" of type "String" in data mapper
```
Comment (avano) [inline ui-tests/src/test/resources/features/integrations/google-sheets.feature]: When

File: ui-tests/src/test/resources/features/integrations/google-sheets.feature [hunk 3]
```diff
@@ -374,9 +374,44 @@ Feature: Google Sheets Connector
...
+      | sheetId       | sourceSheetId        |
+      | dataRange     | basicChart.dataRange   |
+      | domainRange   | basicChart.domainRange |
+    Then click on the "Done" button
+
+    When publish integration
+    And set integration name "charts"
+    And publish integration
+    And navigate to the "Integrations" page
+    And wait until integration "charts" gets into "Running" state
```
Comment (avano) [inline ui-tests/src/test/resources/features/integrations/google-sheets.feature]: Then

File: ui-tests/src/test/resources/features/integrations/google-sheets.feature [hunk 4]
```diff
@@ -374,9 +374,44 @@ Feature: Google Sheets Connector
...
+    Then verify that chart was created
+#
```
Comment (avano) [inline ui-tests/src/test/resources/features/integrations/google-sheets.feature]: remove

**Comment by syndesisqe** [comment]
Build finished. Tests PASSED

**Pull request merged**